src/components/App.tsx

```diff
@@ -3827,6 +3827,9 @@ class App extends React.Component<AppProps, AppState> {
 
   private handleAppOnDrop = async (event: React.DragEvent<HTMLDivElement>) => {
     try {
+      if (this.props.onDrop && (await this.props.onDrop(event))) {
+        return;
+      }
```

Comment:
```suggestion
      if (this.props.onDrop)) {
        await this.props.onDrop(event)
        return;
      }
```
for better readablility